src/layouts/EditHomepage.jsx

```diff
@@ -525,131 +550,62 @@ const EditHomepage = ({ match }) => {
       const idArray = id.split("-")
       const elemType = idArray[0]
 
-      let newSections = []
-      let newErrors = []
+      // The Isomer site can only have 1 resources section in the homepage
+      // Set hasResources to prevent the creation of more resources sections
+      if (elemType === "section" && value === "resources") {
+        setHasResources(true)
+      }
```

Comment:
This doesn't actually fix the issue with the section creation - the existing issue is due to the section creation not properly resetting the state of the selected section after a user clicks, which allows the create to continue to be clicked even if the option currently is the generic selector

Reply:
yea this doesn't. this was part of the original codebase - i just shifted it out to enable the code to be refactored cleanly AHHA